pkg/controller/job/job_controller.go

```diff
@@ -1006,11 +1012,11 @@ func nonIgnoredFailedPodsCount(jobCtx *syncJobCtx, failedPods []*v1.Pod) int {
 	return result
 }
 
-// deleteJobPods deletes the pods, returns the number of successful removals
+// deleteJobPods deletes the pods, returns the number of successful removals of ready pods and total number of successful pod removals
 // and any error.
-func (jm *Controller) deleteJobPods(ctx context.Context, job *batch.Job, jobKey string, pods []*v1.Pod) (int32, error) {
+func (jm *Controller) deleteJobPods(ctx context.Context, job *batch.Job, jobKey string, pods []*v1.Pod) (deletedReady, successfulDeletes int32, err error) {
```

Comment:
```suggestion
func (jm *Controller) deleteJobPods(ctx context.Context, job *batch.Job, jobKey string, pods []*v1.Pod) (int32, int32, error) {
```
I think the `err` was unused?

Reply:
Yes, but either all params are named or all are unnamed params.
I thought it might seem more intuitive if I named them instead of having just 2 int32's in the functon signature, but also the comment explains what the return values are.

Reply:
I would like to minimize the diff, and keep clean ups out of the PR, even if desired in long run